apps/condo/domains/property/schema/Property.js

```diff
@@ -166,7 +166,49 @@ const Property = new GQLListSchema('Property', {
                 },
             },
         },
+        uninhabitedUnitsCount: {
+            schemaDoc: 'A number of non-residential units. Number of parking places for unit.unitType = parking',
+            type: Integer,
+            isRequired: true,
+            defaultValue: 0,
+            hooks: {
+                resolveInput: async ({ operation, existingItem, resolvedData }) => {
+                    let uninhabitedUnitsCount = 0
+                    const getTotalUnitsCount = (map) => {
+                        return get(map, 'parking', [])
+                            .map((section) => get(section, 'floors', [])
+                                .map(floor => get(floor, 'units', []).length)
+                            )
+                            .flat()
+                            .reduce((total, unitsOnFloor) => total + unitsOnFloor, 0)
+
+                    }
+
+                    if (operation === 'create') {
```

Comment:
Did not found test for setting `uninhabitedUnitsCount` in "create" operation

Reply:
@AntonAL done